model/team.go

```diff
@@ -19,7 +19,7 @@ type Team struct {
 	UpdateAt       int64  `json:"update_at"`
 	DeleteAt       int64  `json:"delete_at"`
 	Name           string `json:"name"`
-	Domain         string `json:"domain"`
+	URLId          string `json:"urlid"`
```

Comment:
I don't like the name.  Seems weird to have urlId (We're also inconsistent with how we name Url/URL).  This is really team name and the current team name is the DisplayName.  I think we should bite the bullet and rename so it matches channel.